modules/default/newsfeed/newsfeed.js

```diff
@@ -42,6 +42,8 @@ Module.register("newsfeed", {
 		dangerouslyDisableAutoEscaping: false
 	},
 
+	urlPrefix: location.protocol + "//" + location.host + "/cors?url=",
```

Comment:
so this will always be used even if cors is not necessary?

Reply:
yes, I thought about a config option but as it works so too I decided against.

Reply:
Does that have any downsides? I dont have much knowledge into cors, so I am afraif sites might not be loading if this cors hack is used all of the time?

Reply:
I found no downsides.

I tested with the urls provided in #2840 and with mine, no one did work without the cors prefix. These are not the newsfeed url's but the url's of the articles and they are only used if you want read the full article (which I think is not very often).

Reply:
I would feel better if we had a config option to disable cors so that we could point users that run into an issue to it when the next release is out.

Reply:
so on what level do you see this config option? One option for whole newsfeed module or option for every feed?

And which should be the default? From my side I would enable the cors proxy by default.

Reply:
Yes, I would enable the cors proxy by default too. 
And have the config option per newsfeed, not for the whole module.